engine/apps/oss_installation/views/cloud_users.py

```diff
@@ -60,20 +68,7 @@ def get(self, request):
                 }
             )
 
-        return self.get_paginated_response_with_matched_users_count(response, len(cloud_identities))
-
-    def get_paginated_response_with_matched_users_count(self, data, matched_users_count):
-        return Response(
-            OrderedDict(
-                [
-                    ("count", self.page.paginator.count),
-                    ("matched_users_count", matched_users_count),
-                    ("next", self.get_next_link()),
-                    ("previous", self.get_previous_link()),
-                    ("results", data),
-                ]
-            )
-        )
+        return self.get_paginated_response(data, len(cloud_identities))
```

Comment:
`HundredPageSizePaginator` inherits from `PathPrefixedPagePagination`  and therefore all of these attributes, except for `matched_users_count` would be set. `matched_users_count` is being set above in the new `CloudUsersPagination` paginator class.